metaflow/cli.py

```diff
@@ -145,16 +146,18 @@ def cli(ctx):
     pass
 
 
-@cli.command(help="Check that the flow is valid (default).")
+# D.R.Y and config-drive it, because there may be 2/multiple Linters
+@cli.command(help=str(FLOW_VALIDATORS["validators"][FLOW_VALIDATORS["DEFAULT"]]["help"]["command"]))
 @click.option(
     "--warnings/--no-warnings",
     default=False,
     show_default=True,
-    help="Show all Pylint warnings, not just errors.",
+    help=FLOW_VALIDATORS["validators"][FLOW_VALIDATORS["DEFAULT"]]["help"]["option"],
 )
 @click.pass_obj
 def check(obj, warnings=False):
-    _check(obj.graph, obj.flow, obj.environment, pylint=obj.pylint, warnings=warnings)
+    # is linter being used- we must get names correct/meaningful (+ type annotations)
+    _check(obj.graph, obj.flow, obj.environment, use_linter=obj.use_linter, warnings=warnings)
```

Comment:
@savingoyal this is just "naming things" thingy- `use_linter` is more clarity as a boolean choice than a `linter` which maybe further grokked for being a valid object or null in a never so strongly typed lang like python :)